apps/portals/b2ai.standards/src/pages/StandardsDetailsPage.tsx

```diff
@@ -29,7 +29,7 @@ export const standardsCardSchema: TableToGenericCardMapping = {
   subTitle: 'standardName',
   description: 'description',
   link: 'url',
-  secondaryLabels: ['SDO', 'collections', 'topics'],
+  secondaryLabels: ['SDO', 'collections', 'topics', 'dataTypes'],
```

Comment:
Should this match the changed column name?
```suggestion
  secondaryLabels: ['SDO', 'collections', 'topic', 'dataTypes'],
```

Reply:
Yeah. Thanks